src/components/EnterpriseApp/data/hooks/useEnterpriseCuration.js

```diff
@@ -44,6 +44,24 @@ function useEnterpriseCuration({ enterpriseId, curationTitleForCreation }) {
     }
   }, [enterpriseId]);
 
+  const updateEnterpriseCuration = useCallback(async (options) => {
+    try {
+      const response = await EnterpriseCatalogApiService.updateEnterpriseCurationConfig(
+        enterpriseCuration.uuid,
+        options,
+      );
+      const result = camelCaseObject(response?.data);
+      setEnterpriseCuration(result);
+      return result;
+    } catch (error) {
+      logError(error);
+      setFetchError(error);
+      return error.message;
```

Comment:
[curious] the other functions in this hook  (e.g., `getEnterpriseCuration`) re-throws any caught errors that occur (i.e., `throw error`). I'm wondering why this function return the error instead. Should the error handling in these co-located functions be handled similarly?